### ISSUE TITLE
Custom format

### ISSUE THREAD

**Issue by Alkeryn** [flyingrub/scdl#267]

**Comment by flyingrub** [comment]
There is currently no custom format support, but did you take a look at the `-c` option to skip already downloaded track ?

**Comment by Alkeryn** [comment]
yes but it fails in one of my playlist because a track got deleted by soundcloud
found how to skip it using offset but not that practical

**Comment by flyingrub** [comment]
Could you give me the link of the playlist ?

**Comment by flyingrub** [comment]
Your playlist is private it seems

**Comment by Alkeryn** [comment]
Oh wait
https://soundcloud.com/braun-pierre-louis/sets/fav/s-HE00h

**Comment by flyingrub** [comment]
Mm I checked the soundcloud doc it seems they haven't updated their public API, to reflect this restriction changes ... https://help.soundcloud.com/hc/en-us/articles/115003563948-Can-t-find-a-track-anymore
